src/autoscaler/models/custom_metrics.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+type CustomMetric struct {
+	Name          string  `json:"name"`
+	Type          string  `json:"type"`
```

Comment:
what is type, looks like it is not used when emitting ...

Reply:
As we are only supporting "Gauge" metrics so there is no need to ask user to pass the Metrics Type. Removed the Type info